tests/lib/integration2_test.py

```diff
@@ -14,14 +14,15 @@
 
 from mobly import test_runner
 
-from tests.lib import integration_test
```

Comment:
why is this line changed?
we should strictly do indentation change first, then fix tidbits

Reply:
This line is accidentally changed. I reverted it.